firestore/testapp/README.md

```diff
@@ -71,15 +79,15 @@ Building and Running the testapp
 
 -   Add the following frameworks from the Firebase C++ SDK to the project:
 
-    -   frameworks/ios/universal/firebase.framework
-    -   frameworks/ios/universal/firebase_auth.framework
-    -   frameworks/ios/universal/firebase_firestore.framework
+    -   xcframeworks/firebase.xcframework/ios-arm64_armv7/firebase.framework
+    -   xcframeworks/firebase_firestore.xcframework/ios-arm64_armv7/firebase_firestore.framework
+    -   xcframeworks/firebase_auth.xcframework/ios-arm64_armv7/firebase_auth.framework
     -   You will need to either,
         1.  Check "Copy items if needed" when adding the frameworks, or
         2.  Add the framework path in "Framework Search Paths"
             -   e.g. If you downloaded the Firebase C++ SDK to
                 `/Users/me/firebase_cpp_sdk`, then you would add the path
-                `/Users/me/firebase_cpp_sdk/frameworks/ios/universal`.
+                `/Users/me/firebase_cpp_sdk/xcframeworks/**`.
```

Comment:
Does the double asterisk actually work if used as a path in XCode?

Reply:
yes, there's a `recursive` button in the XCode UI, and when it's selected it automatically changes `path` to `path/**`